CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### v1.5.0 July 9, 2020
+
+* Updated to IAP SDK 1.4.0.
```

Comment:
What happens if a developer is using IAP SDK <= 1.3.0? Assuming they don't care about gift cards.

Reply:
I think that might be problematic since there is a new brand type called SQUARE_GIFT_CARD that would no longer exist.  Our example app would also break since we rely on starting the GiftCardActivity.